DIOP.Analysis.Draft/main.tex

```diff
@@ -0,0 +1,266 @@
+
+\hspace*{1em} The graph of PD554 (getting lost in familiar places) over PZ216 (r years of education) shows a weak relationship between those who get lost/don’t get lost and how many years of education they’ve received, due to how the longer bars that correspond with not getting lost in familiar places are towards the lower years of education. When plotting PD554 against PC273 (ever had dementia), we see the higher frequency of those who responded “yes” to getting lost in familiar places are categorized at 1 on the x-axis, which corresponds to responding “yes” to having dementia. The reverse happens at 5, which is where the respondent is saying they don’t have dementia, and more people are also reporting they aren’t getting lost in familiar places. This provides some insight into a possible correlation, but it’s not strong enough to form a definitive conclusion. However, looking at PV009 (forgetful during daily activities) over PZ216 (r years of education) provides stronger evidence of a relationship. There is a clear steady increase in those who responded “no” to being forgetful as their reported years of education increase. This is our main point of reference for our data and resulting inferences. The interpretation of this in conjunction with our hypothesis hints at our projection being correct due to showing that the more time spent in school decreases the likelihood of developing dementia. When PV009 (forgetful during daily activities) was plotted against PC273 (ever had dementia), we can see that those who responded “yes” to being forgetful during daily activities and also having dementia are the larger spike while at 5 on the x-axis (representing someone who doesn’t have dementia), has the largest spike for not being forgetful. Both of these graphs together support our hypothesis even more as they indicate that the increase in years of schooling stimulates brain activity which further increases cognitive health. In the two cross tabulation tables comparing PC273 (ever had dementia) to PB016 (r college degree) and PB015 (r earned high school diploma/GED) we see a stronger relationship in the PB015 table. On both tables, we focused our attention on 1 (yes) and 5 (no) for ever having dementia, and 1 (yes), 5 (no), and 2 (GED) which only shows up on the PC273 versus PB015 table. The difference in frequency between those who have/do not have dementia and whether or not they have a college degree are quite close in value, which does not indicate a strong correlation to us. However, PC273 compared to PB015 provides a clearer picture of our data in relation to our hypothesis. It is evident that the amount of people who earned a high school diploma or GED have a lower frequency of dementia diagnosis. 
```

Comment:
Your graphs need to not have these numeric labels. They should have labels that give the meaning.

> This provides some insight into a possible correlation, but it’s not strong enough to form a definitive conclusion.

This doesn't speak to your hypothesis. It's just asking, essentially, whether getting lost is a symptom of dementia. I don't think that's an open question that needs to be answered. I encourage you to drop these figures; as you already have a pretty significant imbalance between the number of figures and the explanation of them. Every figure you include should be discussed explicitly.

I can't tell very well if you're already doing that because I'm having a hard time tracking between your text here and the figures. But it seemed like there were way too many figures to be discussed in this one (albeit very long) paragraph. The figures should appear more or less on the same page as the discussion of them--not all bunched together before the discussion. And each paragraph should be focused around one main idea.

> There is a clear steady increase in those who responded “no” to being forgetful as their reported years of education increase.

YES! This is exactly the kind of discussion of the figures that is needed!

> The interpretation of this in conjunction with our hypothesis hints at our projection being correct due to showing that the more time spent in school decreases the likelihood of developing dementia.

Can you improve clarity on long sentences like this one? Maybe break it up or take out unnecessary parts. Something like "This result supports our hypothesis that the likelihood of developing dementia decreases with time spent in school."

When I zoom in on the tables, you appear to have the same table twice. And I can't figure out what's going on in it, so I can't give you good feedback at this point on the discussion of the tables.

This seems to just end abruptly without drawing any kind of overarching conclusion about what's been learned from your analysis. Even if you have a discussion section that goes into more detail, this section still needs something to end it.